docs/PublicationPolicy.md

```diff
@@ -0,0 +1,96 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+
+# Publication policies {#publication_policies}
+
+SpECTRE is an open-source code that is developed and maintained by the
+[Simulating eXtreme Spacetimes (SXS) collaboration](https://black-holes.org). We
+ask all authors of scientific publications that make use of SpECTRE's code or
+data to follow the citation guidelines laid out in the section "\ref citing". An
+additional set of guidelines, laid out in the section "\ref
+sxs_publication_policies" applies to publications by members of the SXS
+collaboration which fall under the [policies of the SXS
+collaboration](https://github.com/sxs-collaboration/WelcomeToSXS/blob/master/SxsPolicies.md).
+
+## Citing SpECTRE {#citing}
+
+Please cite SpECTRE in any publications that make use of its code or data. Cite
+the latest version that you use in your publication. The DOI for this version
+is:
+
+- DOI: [\spectredoi](https://doi.org/\spectredoi)
+
+You can cite this BibTeX entry in your publication:
+
+- [Find BibTeX entry for this version on Zenodo](https://zenodo.org/record/\spectrezenodoid/export/hx)
+
+To aid reproducibility of your scientific results with SpECTRE, we recommend you
+keep track of the version(s) you used and report this information in your
+publication. We also recommend you supply the YAML input files and, if
+appropriate, any additional C++ code you wrote to compile SpECTRE executables as
+supplemental material to the publication.
+
+## Policies that apply to SXS publications {#sxs_publication_policies}
+
+These policies apply to all „science papers” by members of the SXS collaboration
```

Comment:
we should probably use the US-style quotations to be consistent

Reply:
fixed 👍